Resources/Prototypes/Entities/Objects/Materials/parts.yml

```diff
@@ -26,6 +26,9 @@
   name: metal rod
   suffix: Full
   components:
+  - type: Tag # Frontier
+    tags: # Frontier
+    - RodMetal # Frontier
```

Comment:
A sin
How about an empty comp, let me know if you need me to make it.

Reply:
Sure, sounds good. I'll wait for your contribution.

Reply:
Done

Reply:
Nice